src/modules/auth/register/register.screen.jsx

```diff
@@ -1,9 +1,121 @@
-import React, { useEffect } from "react"
-import { useTranslation } from "react-i18next"
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+
+import { Container, FormItem } from "./styled";
+import schema from "./schema";
 
 function RegisterScreen() {
-  const { t } = useTranslation(['common']);
-  return <div>{t('name')}</div>;
-}
+  const { t } = useTranslation(["register"]);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      full_name: "",
+      sex: "1",
+      date: null,
+      email: "",
+      phone: "",
+      password: "",
+    },
+    resolver: yupResolver(schema(t)),
+  });
+  const onSubmitHandler = (data) => {
+    console.log({ data });
+    reset();
+  };
+
+  return (
+    <Container>
+      <h2 className="title">{t("signup")}</h2>
+      <form onSubmit={handleSubmit(onSubmitHandler)}>
+        <FormItem>
+          <label>{t("name")}</label>
+          <input
+            {...register("full_name")}
+            // placeholder="nhap ho ten"
+            name="full_name"
+            required
+          />
+        </FormItem>
+        <p className="errors-message">{errors.full_name?.message}</p>
+
+        <FormItem>
+          <div className="group-button">
+            <label>{t("sex")}</label>
```

Comment:
label cần thuộc tính htmlFor
thường là for tới cái id của thẻ input
<label htmlFor="x"

thì sẽ focus vào
<input id="x"